nvflare/lighter/impl/master_template.yml

```diff
@@ -1551,3 +1557,399 @@ adm_notebook: |
   "nbformat": 4,
   "nbformat_minor": 5
   }
+
+aws_start_svr_sh: |
```

Comment:
Sure, I will put the refactoring as one task in the future.

Reply:
I like the re-factoring idea as well.
It would be better to have smaller files maybe organized in directories instead of having them all in one place.